<!doctype html>
<html lang="en">
<head>

    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta name="description" content="">
    <meta name="author" content="">
    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
    <title>TSL Training | Tools</title>

    <!-- Latest compiled and minified CSS -->
    <link href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.5/flatly/bootstrap.min.css" rel="stylesheet" integrity="sha256-sHwgyDk4CGNYom267UJX364ewnY4Bh55d53pxP5WDug= sha512-mkkeSf+MM3dyMWg3k9hcAttl7IVHe2BA1o/5xKLl4kBaP0bih7Mzz/DBy4y6cNZCHtE2tPgYBYH/KtEjOQYKxA==" crossorigin="anonymous">

    <!--  Latest compiled and minified JavaScript -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
      <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
      <!--[if lt IE 9]>
        <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
        <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
      <![endif]-->
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link href="http://getbootstrap.com/examples/carousel/carousel.css" rel="stylesheet">
</head>
<body style="padding-top:52px;">
  <nav class="navbar navbar-inverse navbar-fixed-top">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">TSLtraining</a>
    </div>
    <div id="navbar" class="collapse navbar-collapse">
      <ul class="nav navbar-nav">
        <li><a href="index.html">Home</a></li>
        <li><a href="courses.html">Upcoming Courses</a></li>
        <li><a href="badges.html">Badges</a></li>
        <li><a href="slurm.html">Cluster</a></li>
        <li><a href="galaxy.html">Galaxy</a></li>
        <li class="active"><a href="tools.html">Tools</a></li> <!-- list of tools -->
        <li><a href="methods.html">Methods</a></li> <!-- SNP, RNAseq, Assembly, CandiSNP etc -->
        <li><a href="contact.html">Contact</a></li>
        <li><a href="about.html">About</a></li>
      </ul>
    </div><!--/.nav-collapse -->
  </div>
</nav>


<div class="container">
    <h1> Tools </h1>
    <h3>Maker – gene calling with aplomb</h3>
    <p>Whatever a plomb is.</p>
    <p>Gene calling is a funny thing. Different programs take very different approaches. Time
        and time again in bioinformatics we come up against a situation where we have to find a
        badly defined object hiding in some messed up dataset. Molecular biology is wonderful in
        that its central object, the gene, is particularly badly defined. What joy comes from trying
         to find genes in draft assemblies then, and every program has a different, specialised approach
         (or heuristic). So which program can we believe if they all give different answers? <a href="http://gmod.org/wiki/MAKER">
        Maker</a>, a program from the GMOD project attempts to answer this question. Maker is a pipeline
           that can merge output from several programs to give a consensus answer and allows you to move
           annotations over from one version of an assembly to another.
        Its a one step annotation pipeline, and it’s available on the TSL cluster. Give it a go.</p>
    <p></p>
    <hr/>
    <a id="genome_browser"><h3>bio-svgenes – publisher proof visualisation of genomic regions</h3></a>
    <p>Genome browsers are wonderful things, but (perhaps surprisingly) there are few that generate
        truly publication quality plots of the data. Most need you to screen-grab or export PNG files that suffer
        from compression or other display problems. A problem noticed by the publication-happy staff of TSL and one
        that the bioinformatics team have been working to address. During her seminar today Diane showed the first
         public display of the in-house created bio-svgenes package. bio-svgenes is a Ruby package that takes in
         pre-prepped data, such as gene models and coverage counts and spits out full Illustrator/Inkscape compatible, true Scalable Vector
        Graphics images that you can send off for publication at any resolution.</p>
    <p>bio-svgenes is relatively easy to use, it hasn’t got a GUI, you need to specify options in a simple config file and
        run a command-line script as a minimum, though for you code-monkeys there is an API, so you can generate all the
        images you need scriptaculously. So if you want nice renderings of BLAST data,
        or some other weird and wonderful use then this is a great package for you.</p>
    <p>Here’s an example of the output, ironically in PNG format because not all browsers can render SVG properly, but its indicative of what you can achieve very quickly.</p>
    <img class="img-responsive" src="http://tsltraining.tsl.ac.uk/wp-content/uploads/2012/10/drawn_from_json2.png" alt="example output"/>
    <p>Don’t worry if you don’t like the colours, its completely flexible and you can set the visual aspects very easily.</p>
    <p>For now bio-svgenes is available on request from me, once the documentation is up to scratch then I’ll make it available
        as a regular gem. If you have no idea what on earth this means and want to give it a go feel free to drop by and ask.</p>
    <p></p>
    <hr/>
    <h3>Circos: Nifty Diagrams – Configuration Hell</h3>
    <p>Diane’s very interesting seminar today contained a wonderful, Nature/Science-style headline grabbing data-vis of many different data-types
        relating to her PST tribes. This great data-ful vis was created by Diane in the Circos software.</p>
        <p>Circos is a pretty neat piece of software designed to create specifically circular diagrams that have a very high ‘data-to-ink’
            ratios. It has gained a lot of ground due to its immense flexibility, you can see it
            in lots of publications (including some authored by TSL members <a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0045307">
                like this recent one</a>.</p>
        <p>Like most truly powerful software, Circos is complex and hard to use. It doesn’t help that it is provided as a Perl
            script. Creating images with Circos requires a fair amount of data pre-prep and configuration tweaking, but the results
             are well worth it. If you’d like to give Circos a go for rendering the results of your projects you can <a href="http://circos.ca">
                 get it here</a>. If you’re having issues with its install and use,
            come and see us down in the bioinformatics office and we’ll see what we can do.</p>
    <p></p>
    <hr/>
    <a id="velvet"><h3>Who would win in a fight between…</h3></a>
    <h4>CLC Bio Assembly Cell and Velvet?</h4>
    <p>It’s a good question.</p>
    <p>Sequence assembly, taking short reads and re-assembling them into long useful contigs
        is the cornerstone of many of our genomics projects. The recent successes of the CLC Bio Assembly
        Cell software has meant that its getting talked up over some of our older methods. At <a href="http://bioinformatics.tsl.ac.uk/clc/clc.html">
        this link</a> (TSL Site only), you’ll see a head-to-head of Velvet and CLC Bio. Who wins? Remember, there are
        no big red buttons and no perfect pieces of software. The result, as ever is mixed.</p>
        <hr/>
  <footer>
    <p>&copy; Team MacLean 2015</p>
    <p class="pull-right"><a href="#">Back to top</a></p>
  </footer>
</div> <!-- /container -->



    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
       <!-- Include all compiled plugins (below), or include individual files as needed -->
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>

</body>
</html>
